wiki/Guides/Audio_Editing/vi.md

```diff
@@ -0,0 +1,147 @@
+
+### Giảm tốc độ bit
+
+*Về thông tin liên quan đến nén nói chung, xem: [Nén tệp](/wiki/Guides/Compressing_files)*
+
+Cài đặt và mở Audacity, sau đó làm theo các bước dưới đây:
+
+1. Mở tệp `.mp3` mà bạn muốn giảm tốc độ bit.
+2. Nhấn `Ctrl` + `Shift` + `E`, sau đó ở chỗ `Save as type:` chọn `MP3 Files`, hoặc:
+   1. Nhấp chuột `File`, sau đó `Export`, rồi đến `Export as MP3`.
+3. Trong phần `Format Options`, thay đổi các thiết lập như sau: 
+   1. `Bit Rate Mode`: `Preset`
+   2. `Quality`: `Medium, 145-185 kbps`
+4. Điều hướng tới nơi mà bạn muốn lưu tệp.
+   - Bạn cũng có thể đổi tên tệp.
+5. Nhấp chuột `Save` và một hộp thoại sẽ xuất hiện để bạn nhập siêu dữ liệu (metadata).
+6. Nhấp chuột `OK` sau khi nhập xong siêu dữ liệu.
+
+### Lặp
+
+Cài đặt và mở Audacity, sau đó làm theo các bước dưới đây:
+
+1. Mở tệp `.mp3` mà bạn muốn chạy lặp.
+2. Kéo thả chuột để bôi đen phần bạn muốn lặp.
+   - Rất hiếm khi bạn muốn lặp từ đầu đến cuối đoạn nhạc.
+   - Đôi khi phần điệp khúc hoặc phần lời lặp sẽ là nơi thực hiện việc lặp tốt nhất.
+3. Nhấn `Ctrl` + `C`, hoặc
+   1. Nhấp `Edit`
+   2. Nhấp `Copy`
+4. Tìm nơi để lặp đoạn mà bạn vừa sao chép.
+   - Đó có thể là chỗ kết thúc của phần bạn vừa bôi đen.
+5. Nhấn `Ctrl` + `V`, hoặc
+   1. Nhấp `Edit`
+   2. Nhấp `Paste`
+6. Phát lại cả bài và đảm bảo rằng đoạn lặp nghe thật hay.
+7. Lặp lại nếu cần thiết.
+8. Nhấn `Ctrl` + `Shift` + `E`, sau đó ở `Save as type:` chọn `MP3 Files`, hoặc:
+   1. Nhấp `File`, rồi `Export`, rồi `Export as MP3`.
+9. Trong phần `Format Options`, thay đổi các thiết lập như sau: 
+   1. `Bit Rate Mode`: `Preset`
+   2. `Quality`: `Medium, 145-185 kbps`
+10. Điều hướng tới nơi mà bạn muốn lưu tệp.
+   - Bạn cũng có thể đổi tên tệp.
+11. Nhấp chuột `Save` và một hộp thoại sẽ xuất hiện để bạn nhập siêu dữ liệu (metadata).
+12. Nhấp chuột `OK` sau khi nhập xong siêu dữ liệu.
+
+### Cắt
+
+Cài đặt và mở Audacity, sau đó làm theo các bước dưới đây:
+
+1. Mở tệp `.mp3` mà bạn muốn cắt.
+2. Kéo thả chuột để bôi đen phần bạn muốn cắt.
+   - Đây sẽ là phần kết thúc (outro) dài ngoằng mà bạn không muốn map.
+3. Nhấn `Delete`.
+4. Kéo thả chuột khoảng 3 đến 5 giây cho tới cuối bài.
+5. Nhấp `Effect`.
+6. Nhấp `Fade Out`.
+7. Nhấn `Ctrl` + `Shift` + `E`, sau đó ở `Save as type:` chọn `MP3 Files`, or:
+   1. Nhấp `File`, rồi `Export`, rồi `Export as MP3`.
+8. Trong phần `Format Options`, thay đổi các thiết lập như sau: 
+   1. `Bit Rate Mode`: `Preset`
+   2. `Quality`: `Medium, 145-185 kbps`
+9. Điều hướng tới nơi mà bạn muốn lưu tệp.
+   - Bạn cũng có thể đổi tên tệp.
+10. Nhấp chuột `Save` và một hộp thoại sẽ xuất hiện để bạn nhập siêu dữ liệu (metadata).
+11. Nhấp chuột `OK` sau khi nhập xong siêu dữ liệu.
+
+## mp3DirectCut
+
+[mp3DirectCut](https://mpesch3.de) là phần mề chỉnh sửa âm thanh miễn phí, có thể sửa tệp MP3 mà không cần mã hoá lại, thường sẽ ngăn ngừa việc giảm chất lượng. Đây là phần mềm được khuyên dùng khi cần nâng/hạ âm lượng hoặc cắt âm thanh.
```

Comment:
Đây là khuyến nghị khi cần tăng/giảm âm lượng hoặc cắt âm thanh.

The guide doesn't recommend the software, it recommends avoid re-encoding when needing to adjust volume or cropping.